src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -4954,7 +4955,46 @@
     public static final int GENERIC_END = GeneratedJavaTokenTypes.GENERIC_END;
 
     /**
-     * Special lambda symbol {@code ->}.
+     * Lambda function "(() -&gt; {})"
+     *
+     * <p>For example: </p>
+     * <pre>
+     *  Arrays.stream(a).map(el -&gt; el);
```

Comment:
The code snippet should be compilable. For example, something like
`IntStream.of(1).forEach(n -> {});`